features/admin.applications.v1/components/try-it/try-it-create-wizard.tsx

```diff
@@ -153,7 +151,21 @@ export const LoginPlaygroundWizard: FunctionComponent<LoginPlaygroundWizardProps
                         <Grid.Row textAlign="center">
                             <Grid.Column stretched width={ "16" }>
                                 <GenericIcon
-                                    icon={ InformationIcon }
+                                    icon={ (
+                                        /* eslint-disable max-len */
+                                        <svg width="80" height="80" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is hardcoding the icon here intentional?

Reply:
<img width="749" alt="Screenshot 2024-08-13 at 10 59 36" src="https://github.com/user-attachments/assets/2bc1bae4-7d81-4fa7-8f41-7fb54a0d6eb6">
This is the icon and there's no point of maintaining this in the `theme` as a re-usable resource.